contracts/VaultV1.sol

```diff
@@ -2,13 +2,14 @@
 pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
 import "./utils/ERC4626.sol";
 import "./interfaces/IStrategy.sol";
 
 // An ERC4626 compliant vault that interacts with a strategy address
 // BUFFER is the minimun amount of tokens that can be stored in the vault and should
 // be compared with bufBal to determine if the vault neeed topup
-contract VaultV1 is ERC4626, Ownable {
+contract VaultV1 is ERC4626, Ownable, AccessControl {
```

Comment:
I don't see why we need access control here? Can we not just use `owner`?

Reply:
We can, was using governance for changing whitelist